snapcraft/cli/assertions.py

```diff
@@ -151,6 +153,45 @@ def edit_collaborators(snap_name, key_name):
             echo.warning("The collaborators for this snap have not been altered.")
 
 
+@assertionscli.command("list-validation-sets")
+@click.option("--name", metavar="<name>", help="Name filter.")
+@click.option("--sequence", metavar="<sequence>", help="Sequence filter.")
```

Comment:
"Filter" suggests we could use something like regexp, glob or ranges (for numeric), but it seems that the dashboard API requires exact matches? Maybe describe as "Validation set name", "Sequence number" instead?

Reply:
Sounds good